charts/httpd/tests/defaults_test.yaml

```diff
@@ -68,17 +40,40 @@ tests:
           path: spec.template.metadata.labels["app.kubernetes.io/instance"]
           value: "RELEASE-NAME"
       - equal:
-          path: spec.template.spec.containers[0].imagePullPolicy
+          path: "spec.template.spec.containers[*].imagePullPolicy"
```

Comment:
Can there be more than one container? If not, I would have kept `0` to be like the other tests below. (nit)

Reply:
Yes there can be multiple containers (`spec.template.spec.containers` is a list).
The test here checks the imagepullpolicy set for any containers, it's the goal.

The test below have a different meaning and only checks the first container: it's not a matter of codestyle of homogeneity, but a matter of intent.